web-app/src/components/ConnectRpcExample.tsx

```diff
@@ -0,0 +1,48 @@
+import { AuthProvider, OpenTDF } from '@opentdf/sdk';
+import { useState } from 'react';
+
+interface ConnectRpcExampleProps {
+  authProvider: AuthProvider;
+}
+
+export function ConnectRpcExample({ authProvider }: ConnectRpcExampleProps) {
+  const [result, setResult] = useState('');
+
+  const client = new OpenTDF({
+    authProvider,
+    platformUrl: '/api',
+  });
+
+  const handleWellknown = async () => {
+    const response = await client.services.v1.wellknown.getWellKnownConfiguration({});
+    setResult(JSON.stringify(response.configuration));
+  };
+
+  const handleKas = async () => {
+    const response = await client.services.v1.access.publicKey({});
+    setResult(response.publicKey);
+  };
+
+  const handlePolicy = async () => {
+    const response = await client.services.v1.attributes.listAttributes({});
+    setResult(response.attributes.map((s) => `${s}`).join(','));
+  };
+
+  return (
+    <>
+      <fieldset>
```

Comment:
Can you add test coverage to this in web-app/tests/tests?

Reply:
I've added  tests for the rpc calls in `/lib/tests/web/platform-rpc.test.ts` and they are run under the wtr.
This component is mostly used for debugging, it handles click to a button and makes an rpc call. The rpc calls are covered by the mentioned test before.